js/interactive-guides/iguide-multipane.js

```diff
@@ -0,0 +1,49 @@
+var currentView;
+var widgetDivs;
+function init() {
+    if (document.body.scrollWidth <= 1170) {
+        currentView = 'single';
+        widgetDivs = $('.stepWidgetContainer');
+    } else {
+        currentView = 'multi';
+        widgetDivs = $('.stepWidgetContainer');
+    }
+}
+
+//TODO: functions for getting (and setting) single/multi pane var?
+//      to be used in other areas of multipane code
+
+$(window).on('load', function() {
+    init();
+
+    //TODO: when switching to multi, determine what step is selected 
+    // and show correct widgets (iguides-common PR #185)
+    $(window).on('resize', function() {
+        if (currentView == 'multi' && window.innerWidth <= 1170) {
+            // move widgets to single column
+            currentView = 'single';
+            console.log('Switch to single column at ', window.innerWidth);
+
+            var widgetDivsArray = [].slice.call( widgetDivs.detach() );
```

Comment:
In this line, I use a quick hack to convert the NodeList returned by JQuery's `detach()` function into a standard array.